app/src/main/java/org/android/go/sopt/extension/CommonExtension.kt

```diff
@@ -0,0 +1,8 @@
+package org.android.go.sopt.extension
+
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
+import okhttp3.RequestBody.Companion.toRequestBody
+
+class CommonExtension {
+    private fun Int.toRequestBody() = toString().toRequestBody("text/plain".toMediaTypeOrNull())
+}
```

Comment:
```suggestion
private fun Int.toRequestBody() = toString().toRequestBody("text/plain".toMediaTypeOrNull())
```

코틀린이니까 파일안에 함수만 있어도 될듯

Reply:
file로 만든게 아니라 class로 만들었었네요 이런 실수를 ㅋㅎㅋㅎㅋㅎ 감사합니다